### PR TITLE
test: bump jest

### PR DESCRIPTION
**Pull request by colincclark**
There are numerous but intermittent CI failures on TC, most likely relating to this recent fix on Jest:

https://github.com/facebook/jest/commit/2ea26852155dbc50b9b4e583259382c4aaea6ec2

* test: disable brightcove tests which is a deprecated package
* add `testURL` which is now a requirement for tests with `jsdom` [see this thread](https://github.com/facebook/jest/issues/6766)
* test: fix test names now that Jest uses test names for snapshots and we lowercase everything
* feat: instrument transformed files in `jest-configurator` now that jest transforms need to provide code coverage
* test: remove custom coverage paths from `jest-configurator` and pass `coveragePathIgnorePatterns` to `instanbul`'s `exclude` instead. Also only collect coverage for the given packages `src` dir.
* fix globs that are now relative to `src`
* chore: bump lock

### CODE REVIEW

File: packages/image/__tests__/web/jest.config.js [hunk 1]
```diff
@@ -2,6 +2,6 @@ const jestConfigurator = require("@times-components/jest-configurator").default;
 const path = require("path");
 
 module.exports = jestConfigurator("web", __dirname, {
-  coverageIgnoreGlobs: ["add-missing-protocol.js"],
+  coverageIgnoreGlobs: ["index.js", "modal-image.web.js"],
```
Comment (colincclark) [inline packages/image/__tests__/web/jest.config.js]: not sure about these being ignored either. Shouldn't the tests cover these for web?

File: packages/jest-configurator/__tests__/source-loader.test.js [hunk 2]
```diff
@@ -1,100 +1,65 @@
...
+      expect(actual).toEqual(fileContents);
     });
 
-    it("should read the respective source file for a module if it belongs to a monorepo package", () => {
-      const srcFilename = "node_modules/@times-components/src/foo/bar.js";
-      const distFilename = "node_modules/@times-components/dist/foo/bar.js";
-      const rawSource = "raw source";
-
-      fs.readFileSync.mockImplementationOnce(() => rawSource);
+    it("should transform the given file", () => {
```
Comment (colincclark) [inline packages/jest-configurator/__tests__/source-loader.test.js]: can we use `iterator` here?

File: packages/jest-configurator/flow-typed/babel-jest.js [hunk 3]
```diff
@@ -12,6 +12,8 @@ declare module "babel-jest" {
     fileData: string,
     filename: string,
     configString: string,
-    options?: {}
+    options: {
+      rootDir: string
```
Comment (colincclark) [inline packages/jest-configurator/flow-typed/babel-jest.js]: thought we were moving towards TypeScript, and away from flow?

File: packages/key-facts/package.json [hunk 4]
```diff
@@ -43,9 +43,10 @@
     "@times-components/test-utils": "0.2.3",
     "@times-components/webpack-configurator": "2.0.0",
     "babel-cli": "6.26.0",
+    "canvas-prebuilt": "1.6.5-prerelease.1",
```
Comment (colincclark) [inline packages/key-facts/package.json]: :( that this is needed. Canvas is not even used (at least directly) in Key Facts...

File: packages/jest-configurator/flow-typed/babel-jest.js (hunk 3 above)
Comment (nehasri89) [reply packages/jest-configurator/flow-typed/babel-jest.js]: have the same question ?

File: packages/image/__tests__/web/jest.config.js (hunk 1 above)
Comment (craigbilner) [reply packages/image/__tests__/web/jest.config.js]: the way coverage is collected it's not very good at simple import/exports

File: packages/jest-configurator/flow-typed/babel-jest.js (hunk 3 above)
Comment (craigbilner) [reply packages/jest-configurator/flow-typed/babel-jest.js]: yep, would rather migrate separately?

File: packages/key-facts/package.json (hunk 4 above)
Comment (craigbilner) [reply packages/key-facts/package.json]: no idea where this came from....removing

File: packages/button/__tests__/android/jest.config.js [hunk 5]
```diff
@@ -1,5 +1,5 @@
 const jestConfigurator = require("@times-components/jest-configurator").default;
 
 module.exports = jestConfigurator("android", __dirname, {
-  coverageIgnoreGlobs: ["shared.js"]
+  coverageIgnoreGlobs: ["utils/index.android.js"]
```
[the remark this answers is not in the document]
Comment (craigbilner) [reply packages/button/__tests__/android/jest.config.js]: came import/export issues as below

File: packages/jest-configurator/flow-typed/babel-jest.js (hunk 3 above)
Comment (nehasri89) [reply packages/jest-configurator/flow-typed/babel-jest.js]: cool

**Pull request merged**